### DISCUSSION TITLE
Q: ESp32

### DISCUSSION THREAD

**Discussion by SpaceTeddy** [emsesp/EMS-ESP32#1044] [General]
hi proddy,

in saw in your dev branch, that you added some code for the ESP32? Did you plan to have a release for that µC type, or it is already working?

thank a lot

chris

**Comment by proddy** [comment]
I have a prototype for ESP32 but its not as stable as the ESP8266. The UART is much more tricky.

**Comment by SpaceTeddy** [comment]
ok, understood.
